### DISCUSSION TITLE
Property 'id' does not exist on type 'User'

### DISCUSSION THREAD

**Discussion by rishi-afk** [payloadcms/payload#534] [Q&A]
When using the req object from the express route handler along with the payload.authenticate middleware, cannot get id from the user object. I tried using PayloadRequest as the type for the req object but I got the following typescript error:

```
No overload matches this call.
  The last overload gave the following error.
    Argument of type '(req: PayloadRequest, res: Response<any, Record<string, any>, number>) => Promise<void>' is not assignable to parameter of type 'Application<Record<string, any>>'.
      Type '(req: PayloadRequest, res: Response<any, Record<string, any>, number>) => Promise<void>' is missing the following properties from type 'Application<Record<string, any>>': init, defaultConfiguration, engine, set, and 61 more.

```
Any idea how to fix this?

**Comment by jmikrut** [comment]
Hey @rishi-afk — where are you importing `PayloadRequest` from?

What version of Express do you have installed in your project?
